src/FFMpeg/Filters/Video/ResizeFilter.php

```diff
@@ -98,8 +98,10 @@ public function apply(Video $video, VideoInterface $format)
         if (null !== $dimensions) {
             $dimensions = $this->getComputedDimensions($dimensions, $format->getModulus());
 
-            $commands[] = '-s';
-            $commands[] = $dimensions->getWidth() . 'x' . $dimensions->getHeight();
+            // Using Filter to have ordering
+            $commands[] = '-vf';
+            $commands[] = '[in]scale=' . $dimensions->getWidth() . ':' . $dimensions->getHeight() . ' [out]';
```

Comment:
`p1` is a custom variable defined by me.
so we can easy to merge multiple filters with the variables p1, p2, p3, p4 .... p100 somethings like this.